lib/easypost/services/beta_rate.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# A Rate service contains all the details about the rate of a Shipment.
```

Comment:
thought: stating that the service contains all the details of XYZ object isn't true. A service just bundles logical functions together. A model on the other hand is the user-facing object that does contain those details. I'd suggest not having these docstrings here on services and instead add them to the models only since that's the user-facing bits. 

This is true of all of them throughout.

Reply:
feedback accepted